libs/sdk-ui-ext/src/internal/components/configurationPanels/WaterfallChartConfigurationPanel.tsx

```diff
@@ -124,12 +124,18 @@ export default class WaterfallChartConfigurationPanel extends BaseChartConfigura
             featureFlags.enableAxisNameViewByTwoAttributes as boolean;
 
         return axes.map((axis: IAxisProperties) => {
+            const isPrimaryAxis =
```

Comment:
please split and avoid nested ternary operators

Reply:
I have fixed this. Thanks!